src/error/metadata/codes/E039.md

```diff
@@ -0,0 +1,3 @@
+This error occurs when using a `contract` command on a non-contract variant.
+
+The variant you're trying to run this operation on isn't a contract variant. Double check the specified `graph@variant` combination is valid and a contract variant.
```

Comment:
Maybe a note here on how you can double check (where is it in Studio, are there other docs we can link out to?)

Reply:
Will update to explicitly note to look at the Studio variant settings page. We don't have a great place in the docs to point to, but I think https://www.apollographql.com/docs/graphos/delivery/contracts/#editing-a-contract is close.